src/wrappers.js

```diff
@@ -1,95 +1,146 @@
 import React, {Component} from 'react'
 import ReactModal from 'react-modal'
+import Octicon from 'react-component-octicons'
 
-class MyListWrapper extends Component {
+
+class HighlightEditListWrapper extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      addWidgetModeActive: false,
+      editMode: false,
+      addWidgetForm: false,
+      editWidgetForm: null,
     }
+    this.renderAddWidgetButton = this.renderAddWidgetButton.bind(this)
+    this.closeForm = this.closeForm.bind(this)
   }
 
-  renderAddWidgetModal() {
+  renderAddWidgetButton() {
     return (
-      <ReactModal className={'edit-widget-modal btn-group card-header'}
-                  isOpen={this.state.addWidgetModeActive}>
-        <form className='widget-modal-form'>
-          <div className='widget-form-group form-group'>
-            WidgetForm
-          </div>
-          <button className={'edit-widget-submit btn btn-primary'} type='submit'>Submit</button>
-        </form>
-      </ReactModal>
+      <button className={'btn btn-info col'} onClick={() => this.setState({
+        addWidgetForm: !this.state.addWidgetForm,
+        editWidgetForm: null,
+      })}>
+        New Widget
+      </button>
     )
   }
 
+  closeForm() {
+    this.setState({
+      addWidgetForm: false,
+      editWidgetForm: null,
+    })
+  }
+
   render() {
+    const { editWidgetForm, addWidgetForm } = this.state
+    const { renderEditWidgetForm, renderNewWidgetForm, renderList } = this.props
     return (
-      <div>
-        <div className={'edit-widget-list-bar btn-group'} role={'group'}>
-          <button className={'btn btn-info'} onClick={this.setState({addWidgetModeActive: true})}>
-            Add a new widget
-          </button>
-          {this.renderAddWidgetModal()}
-        </div>
+      <div className={'col-lg-4 float-right m-8 '}>
+        <p>
+          This list will highlight the widget that is currently being edited. It's defined in HighlightEditListWrapper
```

Comment:
It seems like github is trying to treat the contents of this p tag as javascript and finding no closing quote. I believe this is an error in github's markdown as JSX should treat this like normal html.